recipes/noaa-oisst/meta.yaml

```
@@ -0,0 +1,27 @@
+title: "NOAA Optimum Interpolated SST"
+description: "Analysis-ready Zarr datasets derived from NOAA OISST NetCDF"
+pangeo_forge_version: "0.3.3"
+pangeo_notebook_version: "2021.05.04"
+recipes:
+  - id: noaa-oisst-avhrr-only
+    module: recipe.py  # the module where to find the recipe
+    name: recipe  # the name of the object to import
```

To be consistent with [ADR2](https://github.com/pangeo-forge/roadmap/blob/master/doc/adr/0002-use-meta-yaml-to-track-feedstock-metadata.md#recipes-section) we would have to do


```suggestion
    object: recipe:recipe
```

and remove `module`

---

And maybe that needs to be in quotes to be valid yaml?

---

@rabernat I've updated `pangeo-forge-prefect` to support the module pattern from [ADR2](https://github.com/pangeo-forge/roadmap/blob/master/doc/adr/0002-use-meta-yaml-to-track-feedstock-metadata.md#recipes-section) ref https://github.com/pangeo-forge/pangeo-forge-prefect/pull/7 and updated the `meta.yaml` here so this should be resolved.